resources/js/fileStatus.js

```diff
@@ -0,0 +1,17 @@
+export default function deduceFileStatus(submissionFile, ithenticateDataStatus)
```

Comment:
This is obviously completely up to the plugin creator preferences. Just noting that more common practice that I am aware of is to rarely use default exports. 

Usually its

```
export function deduceFileStatus...
```
with import
```
import {deduceFileStatus} from '...'
```

But this is really not important..